idm_22.2.1_release (#2348)
oracle/docker-images: OracleUnifiedDirectorySM/imagetool/12.2.1.4.0/README.md

> # Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
> ====================================================================================

Reviewer: ```suggestion
# Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
```

> Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
> ====================================================================================

Reviewer: ```suggestion
# Building an Oracle Unified Directory Services Manager Image with Weblogic Image Tool
```

> # Introduction

Reviewer: ```suggestion
## Documentation
```

Author: changed

> For instructions on building an OUDSM image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oudsm/create-or-update-image#create-or-update-an-oracle-unified-directory-services-manager-image-using-the-weblogic-image-tool).

Reviewer: ```suggestion
Detailed instructions on how to [create or update an Oracle Unified Directory Services Manager container image](https://oracle.github.io/fmw-kubernetes/oudsm/create-or-update-image#create-or-update-an-oracle-unified-directory-services-manager-image-using-the-weblogic-image-tool) using the WebLogic Image Tool can be found in the [Oracle Fusion Middleware on Kubernetes](https://oracle.github.io/fmw-kubernetes/) documentation.
```

However, the provided URL returns a 404 error.

Author: The URL link will be available once we have it published. It is still undergoing last minute checks.
